code/modules/antagonists/eldritch_cult/eldritch_effects.dm

```diff
@@ -26,9 +26,11 @@
 	if(!is_in_use)
 		INVOKE_ASYNC(src, .proc/activate , user)
 
-/obj/effect/eldritch/attacked_by(obj/item/I, mob/living/user)
+/obj/effect/eldritch/attackby(obj/item/I, mob/living/user)
 	. = ..()
 	if(istype(I,/obj/item/nullrod))
+		user.say("BEGONE FOUL MAGIKS!!", forced = "nullrod")
```

Comment:
Is this line really necessary? And could you at least spell magic properly?

Reply:
see https://github.com/tgstation/tgstation/blob/master/code/modules/antagonists/cult/runes.dm#L79